lib/peep.rb

```diff
@@ -0,0 +1,27 @@
+require '/Users/ok/Projects/individual-challenges/chitter-challenge-apprenticeships/lib/db_connection.rb'
+require 'time'
+
+class Peep
+  def self.all
+    result = DBConnection.query('chitter', '
+      SELECT * FROM peeps
```

Comment:
Could you have made an instance of the peep class to store this data rather then a hash